src/test/kotlin/no/nav/familie/ef/mottak/config/RestTemplateTestConfig.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.familie.ef.mottak.no.nav.familie.ef.mottak.config
```

Comment:
Hele denne burde kunne erstattes med 
package no.nav.familie.ef.sak.no.nav.familie.ef.sak.config

import no.nav.familie.http.config.INaisProxyCustomizer
import org.springframework.context.annotation.Primary
import org.springframework.stereotype.Component
import org.springframework.web.client.RestTemplate

@Component
@Primary
class NaisNoProxyCustomizer : INaisProxyCustomizer {

    override fun customize(restTemplate: RestTemplate?) {
    }
}

Det eneste du trenger å endre for lokal kjøring er proxy-funksjoonaliteten.

Reply:
Tror ikke det der vil funke, I ApplicationConfig har dere tatt i bruk en egen NaisProxyCustomizer som ikke går via Spring:
https://github.com/navikt/familie-ef-mottak/blob/master/src/main/kotlin/no/nav/familie/ef/mottak/config/ApplicationConfig.kt#L44
https://github.com/navikt/familie-ef-mottak/blob/master/src/main/kotlin/no/nav/familie/ef/mottak/config/NaisProxyCustomizer.kt

Reply:
Du her rett. Jeg har ikke fått fulgt opp endringene i felles. Hvis du sletter restTemplateBuilder i ApplicationConfig og legger til @Import(RestTemplateBuilderBean::class, i toppen så fungerer det å overstyre proxyen.

Reply:
2ee956b

Testet i preprod med søknadsløpet, funker som før :)